rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -1362,18 +1365,18 @@ void saveLockWhitelist() {
                 });
 
             Mockito
-                    .doAnswer((InvocationOnMock invocation) -> {
-                        storageBytesCalls.add(0);
-                        RskAddress contractAddress = invocation.getArgument(0);
-                        DataWord address = invocation.getArgument(1);
-                        byte[] data = invocation.getArgument(2);
-                        // Make sure the bytes are set to the correct address in the repo and that what's saved is what was serialized
-                        assertArrayEquals(Hex.decode("aabbccdd"), contractAddress.getBytes());
-                        Assertions.assertEquals(LOCK_ONE_OFF_WHITELIST_KEY.getKey(), address);
-                        assertArrayEquals(Hex.decode("ccdd"), data);
-                        return null;
-                    })
-                    .when(repositoryMock).addStorageBytes(any(RskAddress.class), eq(LOCK_ONE_OFF_WHITELIST_KEY.getKey()), any(byte[].class));
+                .doAnswer((InvocationOnMock invocation) -> {
+                    storageBytesCalls.add(0);
+                    RskAddress contractAddress = invocation.getArgument(0);
+                    DataWord address = invocation.getArgument(1);
+                    byte[] data = invocation.getArgument(2);
+                    // Make sure the bytes are set to the correct address in the repo and that what's saved is what was serialized
+                    assertArrayEquals(Hex.decode("aabbccdd"), contractAddress.getBytes());
```

Comment:
I thought we were going to change all these instances of `"aabbccdd"`, `"ccdd"`, etc.

Reply:
not all of them, the ones that are still using the bridge storage provider are defined with that mocked address. The change was made for the ones that now use federation storage provider